external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/test/KafkaSpoutTopologyMainNamedTopics.java

```diff
@@ -96,13 +98,6 @@ protected StormTopology getTopologyKafkaSpout() {
         return tp.createTopology();
     }
 
-    public static Func<ConsumerRecord<String, String>, List<Object>> TOPIC_PART_OFF_KEY_VALUE_FUNC = new Func<ConsumerRecord<String, String>, List<Object>>() {
```

Comment:
I am really confused how this could cause serialization problems.  If this really is an issue we need to update the docs too.